.github/workflows/check-semver.yml

```diff
@@ -2,23 +2,23 @@ name: Check semver
 
 on:
   pull_request:
-    types: [opened, synchronize, reopened, ready_for_review]
+    types: [ opened, synchronize, reopened, ready_for_review ]
   workflow_dispatch:
 
 concurrency:
   group: check-semver-${{ github.event.pull_request.number || github.ref }}
   cancel-in-progress: true
 
 env:
-  TOOLCHAIN: nightly-2024-06-01
```

Comment:
We needed to lock this to a specific nightly since the https://github.com/paritytech/parity-publish only supports a specific version.  
Not sure if this will still work now.

Reply:
Hmm so there is a mapping between parity-publish and supported nightly? Disabled merge for now.

Reply:
Yea it uses some docs export feature from nighly, but it reads like the newest version can also handle stable: https://github.com/obi1kenobi/cargo-semver-checks/releases/tag/v0.36.0#:~:text=all%20stable%20Rust%20versions  
Just need to update our parity tool.